packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -192,6 +163,13 @@ export class MongoDBDataProvider<Type = any> implements GraphbackDataProvider<Ty
       }, {});
   }
 
+  private verifyMongoDBPrimaryKey(modelName: string, primaryKeyDesriptor: {name: string, type: string} ) {
```

Comment:
You created an interface for this:

```suggestion
  private verifyMongoDBPrimaryKey(modelName: string, primaryKeyDesriptor: PrimaryKeyDescriptor) {
```

Reply:
Yeap. Good catch!